src/services/users/index.js

```diff
@@ -1,6 +1,5 @@
 const jwt = require("jsonwebtoken")
-const { 
-	genPasswordHash, 
+const {  
```

Comment:
we seem to be having a circular dependency issue for a while and its currently making the reset password test fail. Don't know if its same at your end but not all the tests are passing on this branch though there's nothing wrong with them per se

Been struggling to see the reason but dug a bit to see, its from the `lib/index.js`. 

Kindly take out the `passportjwt` import and export from there

Reason: the passportjwt being exported there makes use of  the `users/index.js` and the `user/index.js` also imports from the `lib/index.js` hence the dependency being circular